backend/functions/lib/adapters/gateways/user-session-cache.repository.ts

```diff
@@ -1,7 +1,7 @@
 import { CamsSession } from '../../../../../common/src/cams/session';
-import { ApplicationContext } from '../types/basic';
 
+// TODO: MOve this into gateway.types.ts
```

Comment:
handle?

Reply:
I agree this should be relocated with the rest of the gateway types.